tests/security_test.json

```diff
@@ -0,0 +1,19 @@
+{
+	"commands": [
+		{
+			"name": "apt-get update",
```

Comment:
I think here we might actually want to move these into one test that does both the apt-get update && apt-get upgrade. You'll probably need to do something like "sh" as the command, and put everything else as a flags list to support the &&.

I think the security test as written was fine, but doing it this way in the yaml introduces a little coupling, the update is really a "prereq" for the upgrade, we don't care about the results at all. Doing it this way makes it so we have to run these in the right order, instead of someday parallelizing these or running them in different instances of the image under test.